### ISSUE TITLE
Docker image for self hosting

### ISSUE THREAD

**Issue by truedem0n** [dunsuite/dunsuite#1]
Hello,

Would it be possible to create a docker image and docker compose for this? so that it's easy to self host?

**Comment by Spikeysanju** [comment]
@truedem0n Yes, we are working on it. We are open-sourcing [DunTasks](dun.sh/tasks) at the end of this month. We will be having a Docker image and a Docker Compose setup come along with the repo.

Will close this issue once we launched the DunTasks.

**Comment by truedem0n** [comment]
any updates?

**Comment by ANorseDude** [comment]
Any updates on this?
